meta-refkit-core/recipes-devtools/python/python3_%.bbappend

```diff
@@ -0,0 +1 @@
+require python-readline.inc
```

Comment:
Well, it's not only readline which is removed... also gdbm and db are not compiled in.